backend/src/process.py

```diff
@@ -146,6 +165,8 @@ async def run(
         items = list(collection)
         length = len(items)
 
+        await self.__update_progress(0, length)
+
```

Comment:
If we do a progress update with `index, length`, then we don't need to do 0% explicitly.

```suggestion
```